### PR TITLE
Release 56.0.0

### PR DESCRIPTION
**Pull request by mcmire**
See changelogs for details.

### CODE REVIEW

**Comment by mcmire** [comment]
@metamaskbot publish-preview

File: packages/assets-controllers/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [Unreleased]
 
+## [9.1.0]
+### Added
+- Add a fifth argument, `source`, to NftController's `addNft` method ([#1417](https://github.com/MetaMask/core/pull/1417))
+  - This argument can be used to specify whether the NFT was detected, added manually, or suggested by a dapp
+
+### Fixed
+- Fix `watchNft` in NftController to ensure that if the network changes before the user accepts the request, the NFT is added to the chain ID and address before the request was initiated ([#1417](https://github.com/MetaMask/core/pull/1417))
+
```
Comment (adonesky1) [inline packages/assets-controllers/CHANGELOG.md:16]: These look good to me!

File: packages/approval-controller/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.0]
+### Added
+- Add `startFlow` and `endFlow` methods to ApprovalController ([#1394](https://github.com/MetaMask/core/pull/1394))
```
Comment (bergarces) [inline packages/approval-controller/CHANGELOG.md:11]: These look good.

File: packages/keyring-controller/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,48 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+    - `addNewAccountWithoutUpdate`
+    - `createNewVaultAndRestore`
+    - `createNewVaultAndKeychain`
+    - `importAccountWithStrategy`
+    - `removeAccount`
+    - `setLocked`
+    - `submitEncryptionKey`
+    - `submitPassword`
+  - The new type omits `vault`, `encryptionKey`, and `encryptionSalt`
+- **BREAKING:** Remove `KeyringState`, `KeyringMemState`, and `KeyringConfig` in favor of new types `KeyringControllerState`, `KeyringControllerActions`, `KeyringControllerEvents`, and `KeyringControllerOptions` ([#1378](https://github.com/MetaMask/core/pull/1378))
```
Comment (Gudahtt) [inline packages/keyring-controller/CHANGELOG.md:37]: The `KeyringMemState` type is still around; it's used as a return value for certain methods

```suggestion
- **BREAKING:** Remove `KeyringState` and `KeyringConfig` in favor of new types `KeyringControllerState`, `KeyringControllerActions`, `KeyringControllerEvents`, and `KeyringControllerOptions` ([#1378](https://github.com/MetaMask/core/pull/1378))
```

Comment (Gudahtt) [reply packages/keyring-controller/CHANGELOG.md:37]: Oh wait, nevermind, it was renamed.

Comment (Gudahtt) [reply packages/keyring-controller/CHANGELOG.md:37]: I guess we should mention the new name though:
```suggestion
- **BREAKING:** Remove `KeyringState`, `KeyringMemState`, and `KeyringConfig` in favor of new types `KeyringControllerState`, `KeyringControllerMemState`, `KeyringControllerActions`, `KeyringControllerEvents`, and `KeyringControllerOptions` ([#1378](https://github.com/MetaMask/core/pull/1378))
```

**Review by Gudahtt — DISMISSED** [review]
LGTM! Just one non-blocking suggestion

**Comment by Gudahtt** [comment]
@SocketSecurity ignore @scure/base@1.1.1
    @SocketSecurity ignore is-plain-obj@4.1.0

These packages don't seem to be unmaintained as the warning suggests; they're just not frequently updated.

**Comment by Gudahtt** [comment]
https://github.com/MetaMask/core/pull/1377 has been merged since this was created, which introduces a number of breaking changes. But I see that those packages aren't touched in this RC, so perhaps we can release this as-is without including those changes. Thoughts?

File: packages/keyring-controller/CHANGELOG.md (hunk 3 above)
Comment (mcmire) [reply packages/keyring-controller/CHANGELOG.md]: You're right, I'm missing KeyringControllerMemState from this list. I had added it to the Added section but that's inconsistent with the other items here. Now that type is only contained in this entry: [0836f80](https://github.com/MetaMask/core/pull/1426/commits/0836f8020c460543ca87f986113f37019dc6ea42)

**Comment by mcmire** [comment]
@Gudahtt Yeah that makes sense not to include those changes here.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**